{"original_position": 36, "diff_hunk": "@@ -218,7 +226,7 @@ decorate_task(\n   AddonStudies.withStudies(),\n   async function testStopNoStudy() {\n     await Assert.rejects(\n-      AddonStudies.stop(\"does-not-exist\"),\n+      AddonStudies.stop(\"does-not-exist\", \"test\"),", "body_text": "I don't really feel all that strongly either way. I think not throwing (and so removing the no-op reasons) might be a slightly safer bet given the few cases in the last 6 months or so where we had issues on startup. It might be nice to have a test that ensures that we end up with the default \"unknown\" value when no reason is passed.", "in_reply_to_id": 160471574, "pull_request_url": "https://api.github.com/repos/mozilla/normandy/pulls/1154", "url": "https://api.github.com/repos/mozilla/normandy/pulls/comments/160531045", "created_at": "2018-01-09T21:34:31Z", "author_association": "CONTRIBUTOR", "body": "I don't really feel all that strongly either way. I think not throwing (and so removing the no-op reasons) might be a slightly safer bet given the few cases in the last 6 months or so where we had issues on startup. It might be nice to have a test that ensures that we end up with the default \"unknown\" value when no reason is passed.", "updated_at": "2018-01-10T19:21:20Z", "html_url": "https://github.com/mozilla/normandy/pull/1154#discussion_r160531045", "pull_request_review_id": 87665380, "_links": {"self": {"href": "https://api.github.com/repos/mozilla/normandy/pulls/comments/160531045"}, "html": {"href": "https://github.com/mozilla/normandy/pull/1154#discussion_r160531045"}, "pull_request": {"href": "https://api.github.com/repos/mozilla/normandy/pulls/1154"}}, "commit_id": "c99552dad03bd2b3d51ba74809410c02dcc93dad", "user": {"following_url": "https://api.github.com/users/gijsk/following{/other_user}", "events_url": "https://api.github.com/users/gijsk/events{/privacy}", "organizations_url": "https://api.github.com/users/gijsk/orgs", "url": "https://api.github.com/users/gijsk", "gists_url": "https://api.github.com/users/gijsk/gists{/gist_id}", "html_url": "https://github.com/gijsk", "subscriptions_url": "https://api.github.com/users/gijsk/subscriptions", "avatar_url": "https://avatars3.githubusercontent.com/u/375983?v=4", "repos_url": "https://api.github.com/users/gijsk/repos", "received_events_url": "https://api.github.com/users/gijsk/received_events", "gravatar_id": "", "starred_url": "https://api.github.com/users/gijsk/starred{/owner}{/repo}", "site_admin": false, "login": "gijsk", "type": "User", "id": 375983, "followers_url": "https://api.github.com/users/gijsk/followers"}, "position": null, "path": "recipe-client-addon/test/browser/browser_AddonStudies.js", "body_html": "<p>I don't really feel all that strongly either way. I think not throwing (and so removing the no-op reasons) might be a slightly safer bet given the few cases in the last 6 months or so where we had issues on startup. It might be nice to have a test that ensures that we end up with the default \"unknown\" value when no reason is passed.</p>", "original_commit_id": "32b3682772ca6e8f579e975d185cfdef91f45388", "id": 160531045}